ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -641,3 +641,35 @@ func TestMemReservationHigherThanMemLimit(t *testing.T) {
 	_, err = ConvertToTaskDefinition(taskDefName, context, serviceConfigs)
 	assert.EqualError(t, err, "mem_limit should not be less than mem_reservation")
 }
+
+func TestSortedGoString(t *testing.T) {
+	family := aws.String("family1")
+	name := aws.String("foo")
+	command := aws.StringSlice([]string{"dark", "side", "of", "the", "moon"})
```

Comment:
Can you change your test case to cover a map instead of a slice?  Slices are always ordered in Go, but maps are traversed in random order.